docs/guides/latest/advanced-use-case/clone-pvc.md

```diff
@@ -259,6 +259,17 @@ We can see from the above output that the backup session has succeeded. This ind
 
 Now, we are going to clone the volumes that we have backed up in the previous section. To do that, we have to create a `RestoreSession` object with `volumeClaimTemplates`.
 
+**Stop Taking Backup of the Old Deployment:**
+
+At first, let's pause the `BackupConfiguration` to stop taking any further backup of the old Deployment so that no backup is taken during the restore process.
```

Comment:
At first, let's pause the scheduled backup of the old Deployment so that no backup is taken during the restore process.